### PR TITLE
Add tests for edge cases in dense packer f-string

### PR DESCRIPTION
**Pull request by thetazero**

### CODE REVIEW

File: tests/test_dense_packer.py [hunk 1]
```diff
@@ -13,8 +13,19 @@ def testFormatter(self):
 
         db = RexDB("test3.db", "cifh?c", lines=1)
         self.assertEqual(db.packer.dense_fstring, "fihcc?")
+
```
Comment (tomyyyD) [inline tests/test_dense_packer.py]: Can you remove the whitespace here for flake8

**Review by tomyyyD — APPROVED** [review]
Just the flake8 thing, but looks good to me.
